helper.py

```diff
@@ -1,41 +1,186 @@
-#!/usr/bin/env python3
+"""
+IMPORTANT NOTE!!
 
-# This file may be obsolete - need to check if scaling in shephard is laggy
+=> The following will ONLY run on Python 3.7 to 3.9
+=> Install websockets version 11.0.3 via pip whl (do not use v12)
+=> Install asynio 0.2.0 via pip whl
+"""
```

Comment:
I'm a bit iffy about blocking our upgrade path. What exactly stops it working on >3.9?

3.9 is already quite old at this point we are on 3.12 now.....

The correct way to version a file in python is to do something like
```py
#!/usr/bin/env python3.7
import sys

major_version, minor_version, micro_version = sys.version_info

if not (major_version = 3 and 7 <= minor_version <= 9):
    raise Error("Bad python version need a version between 3.7-3.9")
```

Reply:
The shebang means if you just run with `python` you might use the right version if your system is setup right